service/src/main/java/uk/nhs/adaptors/gp2gp/common/task/TaskDefinition.java

```diff
@@ -14,6 +14,7 @@
     private final String toAsid;
     private final String fromAsid;
     private final String fromOdsCode;
+    private final String toOdsCode;
```

Comment:
Could we set this new property in EhrExtractRequestHander::createGetGpcStructuredTask and EhrExtractRequestHander::createGetGpcDocumentTask methods?